src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
-body {
+
+@import "./utilites/variables.scss";
+@import "./utilites/mixins.scss";
+@import "./blocks/header.scss";
+@import "./blocks/cards.scss";
+
+* {
   margin: 0;
```

Comment:
don't reset your margins with * selector, use * selector only to set box-sizing

Reply:
Agree, reset margins for tags